### ISSUE TITLE
Connecting lines across columns appear incorrectly sorted.

### ISSUE THREAD

**Issue by andsee** [MindFreeze/ha-sankey-chart#41]
- [Y] I updated to the latest version available
- [Y] I cleared the cache of my browser

Release with the issue: 0.11.0
Browser and Operating System: Windows 11, Chrome and Edge

This could be my configuration but I am unable to figure out why the connecting lines of the children of lounge power jump above items in the 4th column when their entries are after and only present in the 5th column.

The 4 children (AV Left, AV Right, Rear Sonos, Sub) of Lounge power are the last entities in the 5th column and thier parent is the last in the 2nd column.

II also note that the Remaining Switches amount has duplicated it's self magically to the 5th column, which is actually awesome but for some reason The Remaining Misc does not. Perhaps the Switches behavior is a bug, but it would be great to configure this.

Here is my configuration:

`type: custom:sankey-chart
show_names: true
round: 1
height: 400
wide: true
sections:
  - entities:
      - entity_id: sensor.sockets_power
        name: Sockets
        children:
          - sensor.office_power
          - sensor.lounge_power
          - sensor.outlet_outside_power_meter
  - entities:
      - entity_id: sensor.office_power
        name: Office
        children:
          - sensor.outlet_office_bookcase_power_meter
          - sensor.outlet_office_power_meter
          - sensor.outlet_front_xmas_power_meter
          - sensor.outlet_network_current_consumption
      - entity_id: sensor.lounge_power
        children:
          - sensor.outlet_av_left_power_meter
          - sensor.outlet_av_right_power_meter
          - sensor.tapo_2_current_energy
          - sensor.outlet_dining_wall_power_meter
      - entity_id: sensor.outlet_outside_power_meter
        name: Outside
  - entities:
      - entity_id: sensor.outlet_office_bookcase_power_meter
        name: Bookcase
        remaining: Misc
        children:
          - sensor.outlet_window_ups_current_consumption
          - sensor.outlet_dep_pc_current_energy
      - entity_id: sensor.outlet_network_current_consumption
        name: Network
        remaining: Switches
        children:
          - sensor.outlet_office_av_power_meter
  - entities:
      - entity_id: sensor.outlet_window_ups_current_consumption
        name: UPS
        remaining: Modems ?
        children:
          - sensor.outlet_stacker_current_energy
          - sensor.outlet_monitors_current_consumption
  - entities:
      - entity_id: sensor.outlet_stacker_current_energy
        name: PC
      - entity_id: sensor.outlet_monitors_current_consumption
        name: Monitors
      - entity_id: sensor.outlet_dep_pc_current_energy
        name: Desk
      - entity_id: sensor.outlet_office_av_power_meter
        name: NUC
      - entity_id: sensor.outlet_office_power_meter
        name: Office AV
      - entity_id: sensor.outlet_front_xmas_power_meter
        name: Mac Desk
      - entity_id: sensor.outlet_av_left_power_meter
        name: AV Left
      - entity_id: sensor.outlet_av_right_power_meter
        name: AV Right
      - entity_id: sensor.tapo_2_current_energy
        name: Rear Sonos
      - entity_id: sensor.outlet_dining_wall_power_meter
        name: Sub
`

![image](https://user-images.githubusercontent.com/1597978/190103133-044af989-c99f-41bc-b9a7-0f5a20f10ce8.png)

**Comment by MindFreeze** [comment]
The problem here is lack of documentation from me. Haven't found the time. Have a look at the comments in #9 

TLDR: you can add entities with `type: passtrough` in the intermediate sections to control the positions.

**Comment by andsee** [comment]
Thanks I'll give that a go again. I did try it but it got into a mess and I assumed I was using the pass-through incorrectly, probably due to the difficulty of manually ensuring all the children are included and in the right order.

**Comment by MindFreeze** [comment]
@tomlut it has always mattered. There is no sorting currently. You determine the order in the config.

**Comment by tomlut** [comment]
Interesting. I only got crossed charts with the latest update (from 0.11).
